core/src/main/java/com/dtolabs/rundeck/plugins/audit/AuditEventListener.java

```diff
@@ -0,0 +1,58 @@
+package com.dtolabs.rundeck.plugins.audit;
+
+import com.dtolabs.rundeck.core.audit.AuditEvent;
+
+/**
+ * Interface for implementing an Audit event listener plugin.
+ *
+ * @author alberto
+ */
+public interface AuditEventListener {
```

Comment:
it should be named `AuditEventListenerPlugin`

Reply:
I named it that way to allow both use as plugins or internally from within rundeck code. I could maybe do separete interfaces for both cases.

Reply:
Created dedicated interface for plugins.

Reply:
Added a new dedicated interface for plugins.